opslevel/resource_opslevel_user.go

```diff
@@ -91,11 +89,6 @@ func (r *UserResource) Schema(ctx context.Context, req resource.SchemaRequest, r
 					stringvalidator.OneOf(opslevel.AllUserRole...),
 				},
 			},
-			"skip_welcome_email": schema.BoolAttribute{
-				DeprecationMessage: "The skip_welcome_email attribute is deprecated and only kept for backward compatibility.",
```

Comment:
i'm not sure why we put this `DeprecationMessage` message in.  This is a required feature that was built at the request of customers.

Reply:
Since we're keeping this I removed the DeprecationMessage